### PR TITLE
[DBAAS-187] list connections from all accessible namespaces

### PR DESCRIPTION
**Pull request by tchughesiv**
depends on https://github.com/RHEcosystemAppEng/dbaas-dynamic-plugin/pull/40
Screen Shot 2021-11-11 at 9 33 31 AM
Screen Shot 2021-11-11 at 9 35 33 AM

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -68,7 +67,7 @@
   },
   "consolePlugin": {
     "name": "dbaas-dynamic-plugin",
-    "version": "0.1.4",
+    "version": "0.1.3",
```
Comment (yzhao583) [inline package.json:70]: @tchughesiv This comes from my PR maybe, should we make it "0.1.4"?

**Review by yzhao583 — DISMISSED** [review]
lgtm. Thanks!

File: package.json (hunk 1 above)
Comment (tchughesiv) [reply package.json:70]: your call, let's get your PR merged first and then make a decision.

Comment (yzhao583) [reply package.json:70]: @tchughesiv I merged my PR

Comment (tchughesiv) [reply package.json]: thanks, i've rebased

**Review by jeremyary — APPROVED** [review]

**Pull request merged**